src/pages/(dashboard)/profile/(main-profile)/business-card/edit/page.tsx

```diff
@@ -0,0 +1,194 @@
+import { Button, Form } from "@/components/ui";
+import { IconBox } from "@/components/common";
+import { useForm } from "react-hook-form";
+import { ArrowIcon } from "@/components/icons";
+import { Main } from "@/pages/_components";
+import { useState } from "react";
+
+type EditFormValues = {
+	firstname: "";
+	lastname: "";
+	phone: "";
+	email: "";
+	about: "";
+	jobInfo: "";
+	company: "";
+};
+
+const EditProfile = () => {
+	const methods = useForm<EditFormValues>({
+		defaultValues: {
+			firstname: "",
+			lastname: "",
+			phone: "",
+			email: "",
+			about: "",
+			jobInfo: "",
+			company: "",
+		},
+	});
+
+	const { control } = methods;
+	// useState that tracks which dropdown is open. If open it closes by setting to null
+	const [isShow, setIsShow] = useState<string | null>(null);
+
+	const toggleOptions =
+		(showId: string): React.MouseEventHandler<HTMLParagraphElement> =>
+		() => {
+			setIsShow(isShow === showId ? null : showId);
+		};
+
+	return (
+		<Main className="">
+			<section className="relative mt-4 text-center">
```

Comment:
Remove all these back icons, i've already done a general layout for these, so follow the structure in other done pages